nengo_viz/components/action.py

```diff
@@ -126,6 +126,11 @@ def __init__(self, net_graph, uid, type, x, y, width, height, **kwargs):
         cls = getattr(nengo_viz.components, self.type + 'Template')
         self.template = cls(self.obj, **kwargs)
 
+        # Remove any existing sliders associated with the same node
+        for component in self.net_graph.viz.components:
+            if isinstance(component, nengo_viz.components.slider.Slider) and component.node == self.obj: 
```

Comment:
I think, the equals operator `==` should probably be the `is` operator here?

I dislike it, that we have to test for a specific class here. I think, it would be better to introduce an attribute on the components which indicates whether multiple instances are allowed and that we check for that attribute here.